<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget_Bin</class>
 <widget class="QWidget" name="Widget_Bin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>332</width>
    <height>64</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="widget_Bin" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>332</width>
     <height>64</height>
    </rect>
   </property>
   <widget class="QLabel" name="label_Bin">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>20</y>
      <width>91</width>
      <height>26</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>回收站</string>
    </property>
   </widget>
   <widget class="PictureBox" name="binPic" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>26</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PictureBox</class>
   <extends>QWidget</extends>
   <header>picturebox.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
